ckanext/canada/schemas/organization.yaml

```diff
@@ -0,0 +1,145 @@
+scheming_version: 2
+organization_type: organization
+
+about: 'open.canada.ca: Organization metadata definition'
+about_url: http://github.com/open-data/ckanext-canada
+
+form_languages:
+- en
+- fr
+
+#
+#   ORGANIZATION FIELDS
+#
+fields:
+
+# Field = Title.
+# {The organization's title}
+- field_name: title_translated
+  label:
+    en: Title
+    fr: Titre
+  fluent_form_label:
+    en:
+      en: "Title (English)"
+      fr: "Titre (anglais)"
+    fr:
+      en: "Title (French)"
+      fr: "Titre (français)"
+  fluent_help_text:
+    en:
+      en: The English name by which the organization is known
+      fr: pending translation
+    fr:
+      en: The French name by which the organization is known
+      fr: pending translation
+  required: true
+  # copied from fluent_text preset
+  form_snippet: fluent_text.html
+  display_snippet: fluent_text.html
+  error_snippet: fluent_text.html
+  validators: fluent_text unicode scheming_required
+  output_validators: fluent_core_translated_output
+
+# Field = Description.
+# {The organization's description}
+- field_name: notes_translated
+  label:
+    en: Description
+    fr: Description
+  fluent_form_label:
+    en:
+      en: "Description (English)"
+      fr: "Description (anglais)"
+    fr:
+      en: "Description (French)"
+      fr: "Description (français)"
+  fluent_help_text:
+    en:
+      en: A little information about the organization in English
+      fr: pending translation
+    fr:
+      en: A little information about the organization in English
+      fr: pending translation
+  # copied from fluent_markdown preset
+  form_snippet: fluent_markdown.html
+  display_snippet: fluent_markdown.html
+  error_snippet: fluent_text.html
+  validators: fluent_text
+  output_validators: fluent_core_translated_output
+
+# Field = Short Form.
+# {The organization's short form}
+- field_name: shortform_translated
+  label:
+    en: Abbreviation
+    fr: Abréviation
+  fluent_form_label:
+    en:
+      en: "Abbreviation (English)"
+      fr: "Abréviation (anglais)"
+    fr:
+      en: "Abbreviation (French)"
+      fr: "Abréviation (français)"
+  fluent_help_text:
+    en:
+      en: The English abbreviation by which the organization is known
+      fr: pending translation
+    fr:
+      en: The French abbreviation by which the organization is known
+      fr: pending translation
+  required: true
+  # copied from fluent_text preset
+  form_snippet: fluent_text.html
+  display_snippet: fluent_text.html
+  error_snippet: fluent_text.html
+  validators: fluent_text scheming_required
+  output_validators: fluent_core_translated_output
+
+# Field = URL.
+# {The organization's URL}
+- field_name: name
+  label: URL
+  help_text:
+    en: The department identifier used for linking to the department e.g. "tbs-sct" is used to create URL registry.open.canada.ca/en/organization/tbs-sct
```

Comment:
please add `statcan` as an example for a department using a common shortform for both languages